### DISCUSSION TITLE
User Research for Madoc - first steps

### DISCUSSION THREAD

**Discussion by DonnaBaillie** [digirati-co-uk/madoc-platform#446] [General]
Hi all.  Thank you to everyone who I've spoken with so far - much appreciated.  I've put together a proposal for integrating user research into the Madoc programme and welcome your input, comments and help with facilitating this.  Thanks!  Donna

**Madoc User Research proposal**

In order to produce meaningful user research that can inform the ongoing design/refinement process of Madoc, I will need to be able to do the following initially:
•	Identify all relevant user groups 
•	Identify people within these groups to engage in our research – contact lists
•	Identify core user needs for each group
•	Clarify business needs and identify how these intersect with user needs
•	Familiarise myself with a working iteration of Madoc 
•	Establish a clear line of communication with product owner, project manager, developer so that we can schedule usability testing and ensure actions are taken on insights gained via engagement with users

To do this I will need:
•	Client contacts and permission to contact them to arrange initial interviews
•	Access to high level identification of user groups which I believe already exists(?)
•	Access to documentation spelling out the business needs
•	Access to a working iteration of Madoc for my own education
•	Access to either a working iteration of Madoc or wireframes suitable for usability testing with end-users
•	Space on Confluence for Madoc User Research

**Comment by villedigirati** [comment]
Ghent user flows:
https://lucid.app/lucidchart/invitations/accept/inv_37d84856-ba17-422c-8ddc-165bdffc8673

SBB User Needs
https://lucid.app/lucidchart/6e1ea1d5-6dbb-44b5-bf39-bdf431c1824a/view?page=0_0#

**Reply by DonnaBaillie** [reply]
Thanks!

**Comment by christinemacdonald** [comment]
Hi Donna, new space created on Confluence for Madoc https://digirati.atlassian.net/wiki/spaces/MAD/overview?homepageId=3564699807. I've created you a User Research page to do with what you will!! @villedigirati can you please also collate a list of wireframes, Lucid charts etc under the 'UX' page ? I'll add other links to the space as well to various client documentation etc

**Reply by DonnaBaillie** [reply]
Brilliant - thank you!

**Reply by villedigirati** [reply]
I'll repost the original user flows, I don't think the wireframes will be helpful at this stage for Donna (infact it will be confusing).

**Comment by stephenwf** [comment]
Hi @DonnaBaillie I've sent you details for our Madoc dev in Slack.

**Comment by digiratijohn** [comment]
I think that the best way to get started is to focus on active projects to learn both the drivers and the pain points. Once Madoc 2.0 milestone is accomplished there will be plenty to chew on, and the timing will be right for also moving on to wider community engagement of existing and potential customers/user cases. In the meantime we have the Getty as the main active users and maybe SBB could also be spoken to for use cases and feedback. Familiarisation on the product through sitting in on customer meetings, background reading of existing documentation and playing with a demo site will help you build up product knowledge until we have a stable 2.0 release to work with. Suggest we get together with Diego and plan around the above.
